internal/environment/nomad_environment.go

```diff
@@ -80,6 +82,7 @@ func (n *NomadEnvironment) PrewarmingPoolSize() uint {
 }
 
 func (n *NomadEnvironment) SetPrewarmingPoolSize(count uint) {
+	monitoring.ChangedPrewarmingPoolSize(n.ID(), count)
```

Comment:
Is it okay to have the monitoring line here or shall we move it below the next line(s) to ensure we have a valid job config and don't log any wrong data?

Reply:
Since the check in the next lines could just lead to initialization of the data structure and not to an invalidation of the job or the data, I think its makes not difference..

Reply:
Alright, fine for me!